### ISSUE TITLE
pipeline.worker setting requires input to be int

### ISSUE THREAD

**Issue by mattmacnish-nw** [disaster37/terraform-provider-kibana#25]
https://github.com/disaster37/terraform-provider-kibana/blob/df797b8b25676cb2709df031887b68937478d13f/kb/resource_kibana_logstash_pipeline.go#L50

hi, when trying to implement a change to pipeline.worker in logstash pipelines it is not updating correctly,

I believe this is due to the pipeline.workers setting requiring an int rather than a string.

This will happen even with the input being a number in the tf code.

Below is the attached capture of performing this task manually in kibana console:
![image](https://user-images.githubusercontent.com/71022838/151832600-9eccd951-1b78-4920-a5e3-38b1f8f144a7.png)

Below is where the relevant code is passed in TF Plan:
![image](https://user-images.githubusercontent.com/71022838/151832051-215c665a-cb5f-4805-a836-05c0baf769aa.png)

TF-Code referencing pipeline worker:

![image](https://user-images.githubusercontent.com/71022838/151832192-443e8610-f751-4b8a-a972-96c5e641cf0a.png)

**Comment by disaster37** [comment]
Hi,

Can you provide here a terraform sample that look the bug to help me to fix it...

**Comment by disaster37** [comment]
I have tested the following sample, on ELK 8.0.1 and terraform-provider-kibana 8.0.0

```
resource kibana_logstash_pipeline "test" {
    name = "terraform-test"
    description = "test"
    pipeline = "input { stdin{} } output { stdout{} }"
    settings = {
        "queue.type" = "persisted"
        "pipeline.workers" = "8"
    }
}
```

It's work like a charme. After apply it:

```
curl -u elastic:changeme http://kibana:5601/api/logstash/pipeline/terraform-test
{"id":"terraform-test","description":"test","username":"elastic","pipeline":"input { stdin{} } output { stdout{} }","settings":{"pipeline.workers":"8","queue.type":"persisted"}}
```

**Comment by aydasraf** [comment]
Hello @disaster37 .. Thank you for the wonderful work here . I am sorry to tag you here but the issue i have looks relevant to this somehow...   I am having an issue where the `pipeline.workers` & `pipeline.bacth.size`are not honored, I must always modify them manually after. I am working with ELK version `8.2.2` and version `8.0.1` of the provider.. Below is the code snippet i am using 

```HCL
resource "kibana_logstash_pipeline" "logstash" {
  name        = local.pipleine["name"]
  description = "Pipeline for logstash-xxx"
  pipeline    = data.template_file.pipeline.rendered
  settings = {
    "pipeline.batch.size" = "500"
    "pipeline.workers"    = "10"
    "queue.type"          = "memory"
  }
```

however, when deployed the `pipeline.batch.size` is the default of `125` and `pipeline.workers` is the default of `1`

**Comment by olivierots** [comment]
Hello @disaster37 - I reproduced this bug using  ELK 8.0.1 and terraform-provider-kibana 8.0.0
![image](https://user-images.githubusercontent.com/13032092/192493623-45a494c3-007c-42f0-9905-a1de1c25cceb.png)
![image](https://user-images.githubusercontent.com/13032092/192493811-a044143b-18ca-405d-be81-ae699a7895bf.png)

my changes
![image](https://user-images.githubusercontent.com/13032092/192494089-d6707fd2-9638-41b4-9f15-7c2e536a6c43.png)

The pipeline config
![image](https://user-images.githubusercontent.com/13032092/192494548-449d37c4-f370-4a62-85d2-5712ed12ec28.png)

checking the terraform apply for a Logstash pipeline config called "collector"
![image](https://user-images.githubusercontent.com/13032092/192494656-5146e585-c675-4d1e-b864-40630953bf09.png)

The changes applied to the Pipeline Workers setting arent showing though the Pipeline Batch Delay setting change looks OK
![image](https://user-images.githubusercontent.com/13032092/192494893-4ca6fc65-0e17-49e9-8ecf-4cc0b974badc.png)

Could you please advise further ?

**Comment by disaster37** [comment]
Fixed with PR #32 Read PR to handle breaking change.
